about.html

```diff
@@ -0,0 +1,166 @@
+        <li>English</li>
+        <li>My Profile</li>
+        <li>Logout</li>
+      </ul>
+      <header>
+        <span>Zero 2 Hero</span>
+        <nav>
+          <img
+            src="./img/hamburger.png"
+            alt="Hamburger button"
+            class="hamburger"
+          />
+          <img
+            src="./img/close-white.png"
+            alt="navigation close button"
+            class="btn-close"
+          />
+
+          <ul class="mobile-nav">
+            <li class="nav-list">
+              <a href="home.html" class="nav-link">Home</a>
+            </li>
+            <li class="nav-list">
+              <a href="about.html" class="nav-link">About Us</a>
+            </li>
+          </ul>
+
+          <ul class="desktop-nav">
+            <li class="nav-list">
+              <a href="about.html" class="nav-link current">About</a>
+            </li>
+            <li class="nav-list">
+              <a href="#courses" class="nav-link">Courses</a>
+            </li>
+            <li class="nav-list">
+              <a href="#" class="nav-link">Join</a>
+            </li>
+            <li class="nav-list">
+              <a href="#" class="nav-link">Partner</a>
+            </li>
+            <li class="nav-list">
+              <a href="#" class="nav-link">Challenges</a>
+            </li>
+          </ul>
+        </nav>
+        <a href="home.html" class="logo">CHeRos</a>
+      </header>
+
+      <main id="main-container">
+        <section id="about-section">
+          <div class="hero about-hero">
+            <div class="tag-link">
+              <img src="./img/icons8-home-page-24.png" alt="home button" /><img
+                src="./img/icons8-right-chevron-25.png"
+                alt="chevron right button"
+              />
+              <span>About</span>
+            </div>
+            <h2>Hello there!</h2>
+            <h1>Welcome to code heros</h1>
+            <div class="hero-paragraph about-hero-paragraph">
+              <p>
+                At Code Heros, we are here to support people, we are dedicated
+                to helping people grow from zero to heros. A dynamic team
+                positioned to raise problem solvers
+              </p>
+            </div>
+
+            <div class="about-section-action">
+              <p>To find out more about us</p>
+              <a href="#">Follow this link</a>
+            </div>
+          </div>
+
+          <div class="code-challenges">
+            <h2>International coding challenges</h2>
+            <hr />
+            <p>
+              Every year our students participate in international coding
+              challenges that test their coding abilities.
+            </p>
+
+            <div class="challenges">
+              <img src="./img/hackerrank.png" alt="hackerrank logo" />
+              <div class="challenge">
+                <h1>hackerank</h1>
+                <h1>Coding Challenge <br />2024</h1>
+              </div>
```

Comment:
- [ ] Dear @prius2055, throughout your page I can see you have more than 5+ `h1` tags. this is not a good practice. Screen readers and SEO need to focus on a single main content of the page by looking at its `h1` tag. if we have more than one on a single page this confuses the screen reader on where to focus. Please make sure to use only one in each section.

Reply:
Fixed, thanks

Reply:
@prius2055, you still have one extra <h1> on your `index.html` page on line number `136`